### PR TITLE
Input layer test

### PR DESCRIPTION
**Pull request by kamathis4**
Hey @fchollet,
do let me know:-
- Any test/edge cases I have missed

### CODE REVIEW

File: keras_core/layers/core/input_layer_test.py [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+        batch_size = 4
+        input_tensor = np.zeros(input_shape)
+        try:
+            InputLayer(
+                shape=input_shape,
+                batch_size=batch_size,
+                input_tensor=input_tensor,
+            )
+        except ValueError as x:
+            self.assertEqual(
```
Comment (fchollet) [inline keras_core/layers/core/input_layer_test.py:74]: Generally, prefer only matching part of the error message in order to avoid breaking the test every time there's a minor update to the message.

**Review by fchollet — COMMENTED** [review]
Thanks for the PR! Looking good!

**Review by fchollet — APPROVED** [review]
LGTM, thanks!
